discord/guild.py

```diff
@@ -4220,3 +4221,19 @@ async def create_automod_rule(
         )
 
         return AutoModRule(data=data, guild=self, state=self._state)
+
+    async def fetch_onboarding(self) -> Onboarding:
```

Comment:
This feels like odd naming. We use `welcome_screen` to fetch the welcome screen, not `fetch_welcome_screen`. The rules for `fetch_` is if it needs to disambiguate with a corresponding `get_` method but there is no such thing here.